frontend/app-development/features/administration/components/ServiceAdministration.test.tsx

```diff
@@ -14,7 +14,7 @@ import userEvent from '@testing-library/user-event';
 const user = userEvent.setup();
 
 // Mocks:
-jest.mock('react-i18next', () => ({ useTranslation: () => mockUseTranslation() }));
+jest.mock('react-i18next', () => ({ useTranslation: () => mockUseTranslation(), Trans: '' }));
```

Comment:
Should we consider if 
`jest.mock('react-i18next', () => ({ useTranslation: () => mockUseTranslation(), Trans: '' }));` 
should be a part of setupTests.ts as the default setup? 🤔 Then we could avoid having this jest.mocks within every test that needs the react-i18next translate-hook.

Reply:
Good suggestion.

Reply:
I agree, but this is not possible when `mockUseTranslation` depends on some input given in the test file. Overall, I don't think the way we are testing on texts now is ideal. I had some thoughts about this during my holidays, I will share them in Slack so that we can discuss it further.